dnf-plugins-core.spec

```diff
@@ -485,6 +513,7 @@ PYTHONPATH=./plugins nosetests-%{python3_version} -s tests/
 %{python2_sitelib}/dnf-plugins/download.*
 %{python2_sitelib}/dnf-plugins/generate_completion_cache.*
 %{python2_sitelib}/dnf-plugins/needs_restarting.*
+%{python2_sitelib}/dnf-plugins/post-transaction-actions.*
```

Comment:
This line should not be here. It would make post-transaction-actions plugin part of dnf-plugins-core package. We want it as separate subpackage.